internal/client/auth/api_version.go

```diff
@@ -0,0 +1,58 @@
+package auth
```

Comment:
lets not do this - it looks like you've taken loads of upstream code and copied it in here without attribution. 

we'll fix it the right way some other day.